auth/token-managers/iam-token-manager-v1.ts

```diff
@@ -133,24 +137,28 @@ export class IamTokenManagerV1 extends JwtTokenManagerV1 {
     let clientSecret = 'bx';
```

Comment:
We do not want an Authorization header to be sent to the IAM token server in the event that client id/secret are not specified by the user.

Reply:
Oh, so no defaults anymore? Got it. I don't remember that decision. I'll make that change and update the design document just to be clear on it